data/strings/types_strings.txt

```diff
@@ -32412,3 +32412,47 @@
     vi = Không tự phục vụ
     zh-Hans = 无自助服务
     zh-Hant = 沒有自助服務
+
+  [type.amenity.social_facility]
+    en = Social Facility
```

Comment:
I think it would be easier for translators to review the main meaning of the added type. Let's start adding links to the relevant wiki articles.

CC @dwaynebailey 

```suggestion
    comment = https://wiki.openstreetmap.org/wiki/Key:social_facility
    en = Social Facility
```

@organicmaps/translations Please review if the translations below are correct.